namespaces/cloud-platform-live-0.k8s.integration.dsd.io/offender-management-staging/resources/ecr-allocations-public.tf

```diff
@@ -0,0 +1,27 @@
+terraform {
+  backend "s3" {}
+}
+
+provider "aws" {
+  region = "eu-west-1"
+}
+
+module "ecr-repo" {
```

Comment:
You cannot have two modules using the same name. Perhaps you could use something like `ecr-repo-api` and `ecr-repo-public` ? The same applies to the `kubernetes_secret` resource below. Also, don't forget to update the referenced outputs in its `data`!